libs/system-admin-pages/src/pages/blob-explorer-json-file/index.ts

```diff
@@ -0,0 +1,87 @@
+import { requireRole, USER_ROLES } from "@hmcts/auth";
+import { getArtefactMetadata, getJsonContent, getRenderedTemplateUrl } from "@hmcts/publication";
+import "@hmcts/web-core";
+import type { Request, RequestHandler, Response } from "express";
+import "../../types/session.js";
+import { cy } from "./cy.js";
+import { en } from "./en.js";
+
+const getTranslations = (locale: string) => (locale === "cy" ? cy : en);
+
+function formatDateTime(isoString: string): string {
+  const date = new Date(isoString);
+  return date.toLocaleString("en-GB", {
+    day: "2-digit",
+    month: "2-digit",
+    year: "numeric",
+    hour: "2-digit",
+    minute: "2-digit"
+  });
+}
+
+function escapeHtml(text: string): string {
```

Comment:
This is the same as the function in blob-explorer/blob-explorer-publications. Can move to a shared utility function